### ISSUE TITLE
RGBDS Does Not Render Song 

### ISSUE THREAD

**Issue by Saiyaken-PHOENIX** [SuperDisk/UGE#167]
I've had a new issue pop up recently in regards to rendering songs and hUGETracker refusing to give me any files.

-Attempt using RGBDS 0.7.0
Would act as if rendering a piece of music, but does not, no matter where I point the file to.

-Attempt using RGBDS 0.8.0
Would instantly show this crash log upon pressing "Render Song"
```
There was an error assembling the song for playback.

rgbasm output:
rgbasm: unrecognized option: i
FATAL: unknown option '?'
Usage: rgbasm [-EVvw] [-b chars] [-D name[=value]] [-g chars] [-I path]
              [-M depend_file] [-MG] [-MP] [-MT target_file] [-MQ target_file]
              [-o out_file] [-P include_file] [-p pad_value] [-Q precision]
              [-r depth] [-W warning] [-X max_errors] <file>
Useful options:
    -E, --export-all         export all labels
    -M, --dependfile <path>  set the output dependency file
    -o, --output <path>      set the output object file
    -p, --pad-value <value>  set the value to use for `ds'
    -V, --version            print RGBASM version and exit
    -W, --warning <warning>  enable or disable warnings

For help, use `man rgbasm' or go to https://rgbds.gbdev.io/docs/


Please report this issue on the hUGETracker GitHub issues page, and post your song file!

```
[Stats.uge.tar.gz](https://github.com/user-attachments/files/16215607/Stats.uge.tar.gz)

Operating System: Arch Linux 
KDE Plasma Version: 6.1.2
KDE Frameworks Version: 6.3.0
Qt Version: 6.7.2
Kernel Version: 6.9.8-zen1-1-zen (64-bit)
Graphics Platform: Wayland

**Comment by Saiyaken-PHOENIX** [comment]
Update: RGBDS 0.8.0 is completely broken with hUGETracker, I can't playback songs as well.  Going back to RGBDS 0.7.0 remedies this issue but goes back to the problem of songs not rendering at all

**Comment by SuperDisk** [comment]
Did previous versions work for you? Do you have `ffmpeg` installed? If you can preview songs, then rgbds itself is working. I'll take a look soon, thanks for the report.

**Comment by Saiyaken-PHOENIX** [comment]
> Did previous versions work for you? 

The only version of hUGETracker I've had is the latest, at 1.0.8-linux.  It worked before, on both my Arch machines... Something to do with a package update breaking something within ffmpeg?

> Do you have `ffmpeg` installed?

Yes, ffmpeg version n7.0.1

**Comment by SuperDisk** [comment]
Could you try this build? Scroll down to "Linux build" and download it:

https://github.com/SuperDisk/hUGETracker/actions/runs/9930040134

I fixed the RGBDS v0.8.0 issues and this version ships with RGBDS and FFMPEG.

**Comment by Saiyaken-PHOENIX** [comment]
> Could you try this build? Scroll down to "Linux build" and download it:
> 
> https://github.com/SuperDisk/hUGETracker/actions/runs/9930040134


This build works as intended. Thanks.
